exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/DDLTests.kt

```diff
@@ -213,9 +212,6 @@ class DDLTests : DatabaseTestsBase() {
 
         withDb(TestDB.H2) {
             assertEquals("CREATE TABLE IF NOT EXISTS ${"test_named_table".inProperCase()}", testTable.ddl)
-            DMLTestsData.Users.selectAll().where {
-                exists(DMLTestsData.UserData.selectAll().where { DMLTestsData.Users.id eq DMLTestsData.UserData.user_id })
-            }
```

Comment:
Based on the full import and that the query isn't being iterated over or asserted, this seems like an old accidental copy-paste. 
It's also a duplicate of a query in `ExistTests/testExists()`.